unity/Assets/Scripts/ArticulatedArmJointSolver.cs

```diff
@@ -161,9 +162,9 @@ public class ArticulatedArmJointSolver : MonoBehaviour {
                     float distanceRemaining = currentArmMoveParams.distance - distanceMovedSoFar;
 
                     // New version of up-down drive
-                    float forceAppliedFromRest = 500f;
+                    float forceAppliedFromRest = currentArmMoveParams.maxForce;
                     float slowDownTime = 5 * fixedDeltaTime;
```

Comment:
Why do we have this `5` again?

Reply:
morale support!

Reply:
Arbitrary, idea was to give the controller 5 updates to try to slow things down.